src/utils/conditionOperators/DateGreaterThan.js

```diff
@@ -26,7 +26,7 @@ export default class DateGeaterThan extends ConditionOperator {
 
         let conditionTriggerComponent = null;
 
-        if (instance && instance.root) {
+        if (instance?.root?.getComponent) {
             conditionTriggerComponent = instance.root.getComponent(conditionComponentPath);
         }
```

Comment:
You can use optional chaining on [functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) as well.
```
let conditionTriggerComponent = instance?.root?.getComponent?.(conditionalComponentPath)
```